### DISCUSSION TITLE
Gigantic Repo Friendly Output Format

### DISCUSSION THREAD

**Discussion by kuchungmsft** [danielpalme/ReportGenerator#692] [Ideas]
**HtmlInline_AzurePipelines** landing page provides metrics of all assemblies and links to all source files. For gigantic repo, the browser (or maybe the Azure DevOps, not sure which) may not be able to render this landing page. It would be nice to have a new format that
- provides metrics of all assemblies and links to all assembly detail pages (this would be much smaller for the browser to handle), 
- and each assembly detail page then lists links to all source files in that assembly, 
- and source detail page provides the same functionality as of now.

**Comment by danielpalme** [comment]
Thanks for your suggestion.

I will consider this for a new release.
Not sure how I can integrate that into all the different HTML report types. I don't want to duplicate all those report types.
